config/accounts_test.go

```diff
@@ -771,20 +771,34 @@ func TestAlternateBidderCodes_IsValidBidderCode(t *testing.T) {
 			wantErr:     errors.New(`alternateBidderCodes not defined for adapter "pubmatic", rejecting bids for "groupm"`),
 		},
 		{
-			name: "account.alternatebiddercodes config enabled but adapter config has allowedBidderCodes list empty or not defined",
+			name: "account.alternatebiddercodes config enabled but adapter config is disabled",
 			args: args{
 				bidder:          "pubmatic",
 				alternateBidder: "groupm",
 			},
 			fields: fields{
 				Enabled: true,
 				Bidders: map[string]AdapterAlternateBidderCodes{
-					"pubmatic": {},
+					"pubmatic": {Enabled: false},
 				},
 			},
 			wantIsValid: false,
 			wantErr:     errors.New(`alternateBidderCodes disabled for "pubmatic", rejecting bids for "groupm"`),
 		},
+		{
+			name: "account.alternatebiddercodes and adapter config enabled but adapter config has allowedBidderCodes list empty or not defined",
```

Comment:
Can you modify the description here so it doesn't mention an empty list?
`account.alternatebiddercodes and adapter config enabled but adapter config has allowedBidderCodes not defined`

Reply:
done